Packs/BatchData/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "BatchData",
+    "description": "This script is used to split large list data sets into a defined batch size. A common use case is for api requests that can only handle a set number of items per request. ",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "cstone112",
+    "url": "",
+    "email": "",
+    "created": "2022-03-02T16:40:15Z",
+    "categories": [],
```

Comment:
Please supply at least one category, for example: ['Analytics & SIEM', 'Utilities', 'Messaging', 'Endpoint', 'Network Security', 'Vulnerability Management', 'Case Management', 'Forensics & Malware Analysis', 'IT Services', 'Data Enrichment & Threat Intelligence', 'Authentication', 'Database', 'Deception', 'Email Gateway', 'Identity and Access Management', 'File Integrity Management']